### ISSUE TITLE
Is weak-typing supported when modifying record?

### ISSUE THREAD

**Issue by j4nos** [dkharrat/NexusData#11]
In iOS it is possible to modify managedObject like this:
`mo.setValue(obj, forKey: "someAttribute")`

Without strong-typeing it is possible to change record. Is it possible with your solution?

Or only by passing class as argument?
`Task task1 = objCtx.newObject(Task.class);`

**Comment by dkharrat** [comment]
Yes, it's possible. Each `ManagedObject` object exposes `setValue` and `getValue`. For example:

`task1.setValue("name", "the task's name")`

Note that the schema for the ManagedObject must define the field.
